src/Components/Alert/Components/Form/PriceRange.tsx

```diff
@@ -34,7 +33,7 @@ export const PriceRangeFilter: FC<PricaRangeFilterProps> = ({
 
       <PriceRange
         priceRange={state.criteria.priceRange ?? DEFAULT_PRICE_RANGE}
-        onPriceRangeUpdate={handlePriceRangeUpdate}
+        onUpdate={handlePriceRangeUpdate}
```

Comment:
These alert ranges DO debounce updates to the context apparently, so we use the un-debounced callback, so that we don't have stacked debouncing.